public/app/plugins/panel/graph/data_processor.ts

```diff
@@ -119,7 +127,7 @@ export class DataProcessor {
 
     if (datapoints && datapoints.length > 0) {
       const last = datapoints[datapoints.length - 1][1];
-      const from = options.range.from;
+      const from = options.range.from as any;
```

Comment:
`from` is defined as Moment -- but it is often used as a number... I'm not sure if the usage is wrong or the typing is wrong

Reply:
it is always a Moment, so that typing is correct.  The code blow should do from.valueOf()